CHANGELOG.md

```diff
@@ -0,0 +1,427 @@
+- Add peer url to error message in tx5 send (#228) by @matthme in [#228](https://github.com/holochain/kitsune2/pull/228)
+
+### Changed
+
+- Update Cargo.toml use repository instead of homepage (#244) by @szabgab in [#244](https://github.com/holochain/kitsune2/pull/244)
+- Release v0.2.8 by @jost-s in [#238](https://github.com/holochain/kitsune2/pull/238)
+- Check if queue is drained when re-inserting request failed by @jost-s in [#237](https://github.com/holochain/kitsune2/pull/237)
+- Only re-insert requests into queue if their ops are still needed by @jost-s
+- Change mutex unwraps to expects by @jost-s
+- Rectify what get_all_by_key returns by @jost-s
+- Log errors when querying peer meta store for unresponsive urls by @jost-s
+- Simplify flow of outgoing request task by @jost-s
+- Log errors when querying peer meta store for unresponsive urls by @jost-s
+- Check if transport is dropped before checking if url is unresponsive by @jost-s
+- Check that only an unresponsive url's requests are dropped in test by @jost-s
+- Filter peers without url out in filter_unresponsive_agents by @jost-s
+- Rewrite "if x/if not x" to if/else by @jost-s
+- Update comment on unresponsive url by @jost-s
+- Filter unresponsive urls when requesting ops by @jost-s
+- Filter unresponsive urls when publishing by @jost-s
+- Implement PeerMetaStore::get_all_by_key for test store by @jost-s
+- Move long tx5 transport test to integration tests (#236) by @matthme in [#236](https://github.com/holochain/kitsune2/pull/236)
+- Record peer connection failures in the peer meta store (#233) by @matthme in [#233](https://github.com/holochain/kitsune2/pull/233)
+- Modify get/set to mark peer unresponsive in peer meta store by @jost-s in [#234](https://github.com/holochain/kitsune2/pull/234)
+- Bump johnwason/vcpkg-action from 6 to 7 (#232) by @dependabot[bot] in [#232](https://github.com/holochain/kitsune2/pull/232)
+- Faster initiate (#230) by @ThetaSinner in [#230](https://github.com/holochain/kitsune2/pull/230)
+- Notify on fetch queue drained (#227) by @ThetaSinner in [#227](https://github.com/holochain/kitsune2/pull/227)
+- Implement the showcase demo app (#219) by @cdunster in [#219](https://github.com/holochain/kitsune2/pull/219)
+- Permit receiving a burst of gossip initiations (#226) by @ThetaSinner in [#226](https://github.com/holochain/kitsune2/pull/226)
+
+### Fixed
+
+- Send queue drained notification when no requests to be sent is empty by @jost-s
+- Change timeout to 100 ms in unresponsive_urls_are_filtered by @jost-s
+
+### Removed
+
+- Remove only current request from state when url unresponsive by @jost-s
+- Remove backoff and replace with unresponsive function in peer meta store by @jost-s
+
+## First-time Contributors
+
+* @szabgab made their first contribution in [#244](https://github.com/holochain/kitsune2/pull/244)
+
+## [0.2.7] - 2025-05-23
```

Comment:
No 0.2.8 between 0.2.7 and 0.3.0?

Looking above I'd actually expect everything except "Update Cargo.toml use repository instead of homepage" to already be released. I really like the changelog but I'm thinking that it should go up to 0.2.8 and then we don't do a release just yet? As we've already released the breaking changes on 0.2.x, the reason to bump to 0.3.x has probably passed? Perhaps we shouldn't break Holochain dev versions and should be releasing 0.2.X-dev.Y versions until we are ready to release Kitsune2 0.2...